java/main/com/google/privacy/differentialprivacy/Noise.java

```diff
@@ -31,6 +31,14 @@
   long addNoise(
       long x, int l0Sensitivity, long lInfSensitivity, double epsilon, @Nullable Double delta);
 
+  ConfidenceInterval<Long> getConfidenceIntervalInt(
+          long noisedValue, long l0Sensitivity, long lInfSensitivity,
+          double epsilon, @Nullable Double delta, double confidenceLevel);
+
+  ConfidenceInterval<Double> getConfidenceIntervalDouble(
```

Comment:
nit: there should be no need for "Double" or "Int" at the end of the method name. Both methods can be called "getConfidenceInterval".

Reply:
I overloaded the functions similar to addNoise(..).